File: Java/Account.java
public class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    /*public Account(String name, String document){
        this.name = name;
        this.document = document;
    }*/

    public Account(String name, String document, String email, String password){
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }

    void printDataUser(){
        System.out.println("Name: " + name + " Document: " + document + " Email: " + email + " Password: " + password);
    }

    void printDataDriver(){
        System.out.println("Driver: " + name + " Document: " + document + " Email: " + email + " Password: " + password);
    }
}

File: PHP/Account.php
<?php
class Account{
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($name, $document, $email, $password){
        $this->name = $name;
        $this->document = $document;
        $this->email = $email;
        $this->password = $password;
    }

    function printDataUser(){
        echo "Name: $this->name, Document: $this->document, Email: $this->email, Password: $this->password ";
    }

    function printDataDriver(){
        echo "Driver: $this->name, Document: $this->document, Email: $this->email, Password: $this->password ";
    }
}
?>
